pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -90,16 +90,15 @@ func createGitHubInterceptor(secretName, ns string) *triggersv1.EventInterceptor
 	}
 }
 
-func CreateListenerTrigger(name, filter, repoName, binding, template, secretName, secretNS string) triggersv1.EventListenerTrigger {
+func CreateListenerTrigger(name, filter, repoName, template, secretName, secretNS string, bindings []string) triggersv1.EventListenerTrigger {
```

Comment:
We could maybe keep the bindings and template together?
Either move template to just before the binding, or split the declaration.

Reply:
moved template just before bindings